commune.lic

```diff
@@ -0,0 +1,412 @@
+=begin
+  Documentation: ???
```

Comment:
https://elanthipedia.play.net/Lich_script_repository#commune

Just make sure to update elanthipedia once this is merged.